src/pharmpy/tools/amd/run.py

```diff
@@ -118,6 +119,9 @@ def run_amd(
     mechanistic_covariates : list
         List of covariates to run in a separate proioritized covsearch run. The effects are extracted
         from the search space for covsearch
+    perform_retries: str
+        Weither or not to run retries tool. Valid options are 'skip', 'all_final' or 'amd_final'.
+        Default is 'amd_final'.
```

Comment:
I think that what you call `amd_final` should be named `final`. Then the same option can be reused for other tools. `all_final` should perhaps be named `subtool_final` or `selected` or something along those lines.